packages/rrweb/test/replayer.test.ts

```diff
@@ -718,7 +735,7 @@ describe('replayer', function () {
       const replayer = new Replayer(events);
       replayer.play();
     `);
-    await page.waitForTimeout(50);
+    await page.waitForTimeout(150);
```

Comment:
This is a more reliable way of forcing a wait
```suggestion
    await waitForRAF(page);
```

Reply:
updated and thank you!